modules/oadp-storage-class-mapping.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+[id="oadp-storage-class-mapping_{context}"]
+:_mod-docs-content-type: CONCEPT
+= Storage class mapping
+
+Storage class mapping allows you to define rules or policies specifying which storage class should be applied to different types of data. This feature automates the process of determining storage classes based on access frequency, data importance, and cost considerations. It optimizes storage efficiency and cost-effectiveness by ensuring that data is stored in the most suitable storage class for its characteristics and usage patterns.
+
+You can use the `change-storage-class-config` field to change the storage class of your data objects. This feature allows you to optimize costs and performance by moving data between different storage tiers, such as from standard to archival storage, based on your needs and access patterns.
+
+[id=storage-class-mapping-mtc_{context}]
+== Storage class mapping with MTC
```

Comment:
Should this be an H3? Seems like a type of Storage Mapping.

Reply:
It's presently a level 3. When I made the change you requested I received an error message.